cmd/cli/serve/serve.go

```diff
@@ -235,6 +241,7 @@ func serve(cmd *cobra.Command) error {
 		IsComputeNode:         isComputeNode,
 		IsRequesterNode:       isRequesterNode,
 		Labels:                getNodeLabels(autoLabel),
+		WebUI:			   	   startWebUI,
```

Comment:
More wonky

Reply:
why is lint not complaining?

Reply:
I would guess because the value on the struct is being used since we write to it